src/Metable.php

```diff
@@ -855,7 +855,8 @@ protected function castMetaValue(string $key, mixed $value, string $cast): mixed
             return $this->castMetaToCollection($cast, $value);
         }
 
-        if (class_exists($cast)
+        if (
+            class_exists($cast)
```

Comment:
Sorry it took me a while to get back to this.

nit: no newline after opening `if (`